site/book/05-developing-functions/03-developing-in-Typescript.md

```diff
@@ -95,7 +95,7 @@ Currently supported platforms: amd64 Linux/Mac
 
    ```shell
    $ kpt fn source $CONFIGS \
-     | node dist/label_namespace_run.js -d label_name=color -d label_value=orange \
+     | kpt fn eval - --exec "node dist/label_namespace_run.js -d label_name=color -d label_value=orange" \
```

Comment:
label_name and label_value can be provided as function-args..right ?

`kpt fn eval - --exec "node dist/label_namespace_run.ts" -- label_name=color -d label_value=orange` ?

Reply:
No, the arguments after `--` will be converted to functionConfig. Although that depends on function implementation, not sure how `label_namespace_run.js` handle it.

Reply:
Looks like TS SDK will convert the arguments specified by `-d` to a functionConfig. So yes these 2 arguments can be put after `--`. @frankfarzan Do you want to go with this way?

Reply:
Yes

Reply:
You also don't need the source or sink command, just eval.

Reply:
done